Makefile

```diff
@@ -54,6 +54,9 @@ test-local: install  ## Run tests against a localhost API (needs GROUNDLIGHT_API
 test-integ: install  ## Run tests against the integ API server (needs GROUNDLIGHT_API_TOKEN)
 	GROUNDLIGHT_ENDPOINT="https://api.integ.groundlight.ai/" ${PYTEST} ${TEST_ARGS} ${CLOUD_FILTERS} test
 
+test-dev: install ## Run tests against a dev API server (needs GROUNDLIGHT_API_TOKEN and properly configured dns-hostmap)
```

Comment:
Added this because it seems like it might be useful for the future, but let me know if we don't want this in here.

Reply:
I'm indifferent on this one. Only thought I have is that given the above two commands you could instead make these commands set a variable and then call `make test` using that variable

Reply:
Good idea, I made some changes accordingly - let me know if those are what you were envisioning.